query_maven_metadata.py

```diff
@@ -61,6 +61,8 @@ def _parse_arguments(args):
     # this is experimental and not generalized yet
     parser.add_argument("--filter", type=str, required=False,
         help="Generic query filter, currently only supported for artifact queries")
+    parser.add_argument("--force", required=False, action='store_true',
+        help="If set, always generated poms, regardless of whether an artifact has changed since it was last released")
```

Comment:
@dylanShark this help msg doesn't make sense, since query doesn't generate poms.  Could you rephrase please?

Reply:
@simontoens I will update it, sorry I missed the help message.